server/models/flightSchema.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+
+const flightSchema = new mongoose.Schema(
+  {
+    departure: {
+      type: String,
```

Comment:
Or use mongoose.Types.ObjectId as for 'airplane' field or user string for 'airplane'. I like string more